infrastructure/cdn-in-a-box/cache/Dockerfile

```diff
@@ -29,7 +29,7 @@ EXPOSE 80
 ADD https://ci.trafficserver.apache.org/RPMS/CentOS7/trafficserver-7.1.4-2.el7.x86_64.rpm /trafficserver.rpm
 ADD https://ci.trafficserver.apache.org/RPMS/CentOS7/trafficserver-devel-7.1.4-2.el7.x86_64.rpm /trafficserver-devel.rpm
 
-RUN yum install -y bind-utils kyotocabinet-libs epel-release initscripts iproute net-tools nmap-ncat gettext autoconf automake libtool gcc-c++ cronie glibc-devel openssl-devel
+RUN yum install -y bind-utils kyotocabinet-libs epel-release initscripts iproute net-tools nmap-ncat gettext autoconf automake libtool gcc-c++ cronie glibc-devel openssl-devel 'perl(JSON)'
```

Comment:
Do we not need `perl(JSON)`?

Reply:
The RPM spec requires it. If you don't include it, you get
```
 > [20/20] RUN rpm -Uvh /$(basename infrastructure/cdn-in-a-box/cache/traffic_ops_ort.rpm) &&    rm /$(basename infrastructure/cdn-in-a-box/cache/traffic_ops_ort.rpm):
#26 0.548 error: Failed dependencies:
#26 0.548       perl(JSON) is needed by traffic_ops_ort-4.2.0-10848.7af9dd63.el7.x86_64
```

I agree that we don't really need it. But since the other dependency validation is still useful, maybe we should remove that dependency from the spec when we make the project use the go version of ORT that was added in #4872.

Either way, `perl(JSON)` was already being installed, its installation was just deferred until we installed the ORT RPM.

Reply:
well, that's a dependency for the Perl script that's unused. Is there a `--no-dependencies` option or something for `rpm`?

Reply:
The full dependency list for EL7 is

* perl-JSON
* perl-libwww-perl
* perl-Crypt-SSLeay
* perl-LWP-Protocol-https
* perl-Digest-SHA

So, we wouldn't be missing anything by skipping dependency verification.

Dependencies skipped in af14af15b.